astropy/cosmology/flrw/tests/test_w0cdm.py

```diff
@@ -158,13 +155,11 @@ def setup_class(self):
     def test_repr(self, cosmo_cls, cosmo):
         """Test method ``.__repr__()``."""
         super().test_repr(cosmo_cls, cosmo)
-
-        expected = (
```

Comment:
Why did you bother introducing `expected` for a couple of commits? This just clutters the diff.

Reply:
Removed in the git history.